openslides/__main__.py

```diff
@@ -98,6 +98,14 @@ def get_parser():
         action='store_true',
         help='Do not launch the default web browser.')
     subcommand_start.add_argument(
+        '--host',
+        action='store',
+        help='IP address to listen on. Default is 0.0.0.0.')
+    subcommand_start.add_argument(
+        '--port',
+        action='store',
+        help='Port to listen on. Default is 8000.')
```

Comment:
You can use `default` here: https://docs.python.org/3.5/library/argparse.html#default so you don't need the if clausees below.